visidata/features/reload_every.py

```diff
@@ -1,14 +1,17 @@
 import os
 import time
 
-from visidata import vd, BaseSheet, Sheet, asyncthread, Path, ScopedSetattr
+from visidata import vd, BaseSheet, Sheet, asyncignore, asyncthread, Path, ScopedSetattr
 
 
 @BaseSheet.api
-@asyncthread
+@asyncignore
```

Comment:
Hm, the only way to kill this thread would be to quit the sheet.  What if it gets run twice on the same sheet?